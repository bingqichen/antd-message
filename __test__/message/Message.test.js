import React from 'react';
import Message from '../../src/message';

describe('Message', () => {
  it('Component should be render', () => {
    Message.success('成功');
    expect(document.querySelectorAll('.component-message-wrap')[0]).toMatchSnapshot();
  });
});
