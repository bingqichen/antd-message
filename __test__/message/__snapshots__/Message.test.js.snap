// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message Component should be render 1`] = `
<div
  class="component-message-wrap"
  style="animation-duration: 3000ms"
>
  <div
    class="message-content success"
  >
    <i
      class="component-icon-wrap icon-success"
    />
    <p>
      成功
    </p>
  </div>
</div>
`;
